src/components/triggers-list/EventListenersRow.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import {
+  getGroupVersionKindForModel,
+  K8sResourceCommon,
+  ResourceLink,
+  RowProps,
+  TableData,
+  Timestamp,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { useTranslation } from 'react-i18next';
+import { KEBAB_BUTTON_ID } from '../../consts';
+import {
+  Dropdown,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+} from '@patternfly/react-core';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import { K8sCommonKebabMenu } from '../utils/k8s-common-kebab-menu';
+import { getResourceModelFromBindingKind } from '../utils/pipeline-augment';
+
+type EventListenersKebabProps = {
+  obj: K8sResourceCommon;
+};
+
+getResourceModelFromBindingKind;
+
+const EventListenersKebab: React.FC<EventListenersKebabProps> = ({ obj }) => {
+  const model = getResourceModelFromBindingKind(obj?.kind);
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const onToggle = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = () => {
+    setIsOpen(false);
+  };
+
+  const dropdownItems = K8sCommonKebabMenu(obj, model);
+
```

Comment:
<img width="1439" alt="Screenshot 2024-03-25 at 12 43 00 PM" src="https://github.com/openshift-pipelines/console-plugin/assets/102503482/b5213825-648c-48ce-818a-54051ca0f10d">

In TriggerTemplate list page edit and delete button in kebab menu showing TriggerBinding kind